utils/markdown.ts

```diff
@@ -0,0 +1,31 @@
+// https://stackoverflow.com/questions/24686588/how-to-test-if-a-string-has-markdown-in-it
+const isMarkdownString = (text: string): boolean => {
+  return (
+    /\[[^]]+\]\(https?:\/\/\S+\)/.test(text) ||
+    /\s(__|\*\*)(?!\s)(.(?!\1))+(?!\s(?=\1))/.test(text)
+  )
+}
+
+export const convertMarkdownToText = (text = ''): string => {
+  if (isMarkdownString(text)) {
+    text = text.replaceAll('**', '')
+    text = text.replaceAll('__', '')
+    text = text.replaceAll('\n', ' ')
+    text = text.replaceAll('\n', ' ')
+
+    const urlsMatch = text.match(/\[\S+\]\(https?:\/\/\S+\)/g)
+    if (urlsMatch) {
+      urlsMatch.forEach((match) => {
+        const matchName = match.match(/\[\S+\]/)
+        if (matchName) {
+          text = text.replaceAll(
+            match,
+            matchName[0].slice(1, matchName[0].length - 1)
```

Comment:
Not sure what are the requirements
But makes more sense to me to keep the url rather then the display name

Reply:
I think the name of the URL would be more readable while sharing the content, at least for this case

![image](https://github.com/kodadot/nft-gallery/assets/31397967/2aba298b-46df-49c8-9e7f-927f6abc27d2)